xdsl/transforms/lower_riscv_func.py

```diff
@@ -59,9 +59,80 @@ def match_and_rewrite(self, op: riscv_func.SyscallOp, rewriter: PatternRewriter)
         rewriter.replace_matched_op(ops, new_results=new_results)
 
 
+SCALL_EXIT = 93
+"""
+93 is the number of the `exit` syscall on RISCV.
+"""
+
+
+class LowerRISCVFuncOp(RewritePattern):
+    """
+    Temporary lowering of only the main function, would like to lower non-main functions
```

Comment:
I think it would be good to make an issue for this?

Reply:
I think this comment is out of date?

Reply:
Yep definitely out of date, should be removed